hack/jenkins/test-image/run.sh

```diff
@@ -42,6 +38,10 @@ export KUBE_COVERPROCS=8
 export KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
```

Comment:
GitHub won't let me add a comment on the correct lines, but we don't need export KUBE_GOVERALLS_BIN="${GOPATH}/bin/goveralls" or export KUBE_COVERPROCS=8